rasa/core/validator.py

```diff
@@ -22,17 +23,15 @@ def __init__(self, domain: Domain, intents: TrainingData, stories: List[StorySte
         self.stories = stories
 
     @classmethod
-    async def from_files(
-        cls, domain_file: Text, nlu_data: Text, story_data: Text
-    ) -> "Validator":
+    async def from_importer(cls, importer: TrainingFileImporter) -> "Validator":
         """Create an instance from the domain, nlu and story files."""
 
-        domain = Domain.load(domain_file)
+        domain = await importer.get_domain()
         asyncio.new_event_loop()
```

Comment:
Not part of the PR: But why is that line here?

Reply:
threw it out